packages/utils/src/common/search-service.ts

```diff
@@ -0,0 +1,118 @@
+export interface ISearchService {
```

Comment:
comment: In #3510, this lives in `@medusajs/types` - not utils, as it is an interface - and is placed within a domain-specific folder. Will you give that PR a review?

Reply:
+1